tethys_gizmos/static/tethys_gizmos/css/jobs_table.css

```diff
@@ -1,3 +1,11 @@
+.jobs-table, .dataTables_wrapper{
+    margin-bottom: 250px;
+}
+
+.jobs-table th{
+    white-space: nowrap;
+}
+
```

Comment:
```suggestion
.jobs-table>tbody>tr>td {
    vertical-align: middle;
}
```

Before:
![image](https://user-images.githubusercontent.com/5123221/98181408-72973b80-1ec0-11eb-94c6-0ff3c997e775.png)


After:
![image](https://user-images.githubusercontent.com/5123221/98181394-5f846b80-1ec0-11eb-936a-76348ff29089.png)